File: src/main/java/chloe/movietalk/repository/RefreshRepository.java
package chloe.movietalk.repository;

import chloe.movietalk.domain.Refresh;
import org.springframework.data.repository.CrudRepository;

import java.util.Optional;
import java.util.UUID;

public interface RefreshRepository extends CrudRepository<Refresh, UUID> {
    Optional<Refresh> findByUserId(UUID id);
}

File: src/main/java/chloe/movietalk/repository/RefreshRepository.kt
package chloe.movietalk.repository

import chloe.movietalk.domain.Refresh
import org.springframework.data.repository.CrudRepository
import java.util.*

interface RefreshRepository : CrudRepository<Refresh?, UUID?> {
    fun findByUserId(id: UUID?): Optional<Refresh?>?
}
